components/odh-notebook-controller/controllers/notebook_webhook.go

```diff
@@ -253,8 +265,103 @@ func (w *NotebookWebhook) Handle(ctx context.Context, req admission.Request) adm
 	return admission.PatchResponseFromRaw(req.Object.Raw, marshaledNotebook)
 }
 
+func (w *NotebookWebhook) ClusterWideProxyIsEnabled() bool {
+	proxyResourceList := &configv1.ProxyList{}
+	err := w.Client.List(context.TODO(), proxyResourceList)
+	if err != nil {
+		return false
+	}
+
+	for _, proxy := range proxyResourceList.Items {
+		if proxy.Name == "cluster" {
+			if proxy.Spec.HTTPProxy != "" && proxy.Spec.HTTPSProxy != "" &&
+				proxy.Spec.NoProxy != "" && proxy.Spec.TrustedCA.Name != "" {
+				return true
```

Comment:
For non-ssl proxy servers, TrustedCA should not be set. so it should check the TrustedCA value independently.